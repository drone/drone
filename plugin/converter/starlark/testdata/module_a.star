load("testdata/module_b.star", "pipeline_b")

def main(ctx):
  return [{
    'kind': 'pipeline',
    'type': 'docker',
    'name': 'default'
  }, pipeline_b(ctx)] 