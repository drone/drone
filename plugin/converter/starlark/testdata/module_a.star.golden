---
{"kind": "pipeline", "type": "docker", "name": "default"}
---
{"kind": "pipeline", "type": "docker", "name": "default"}
